XenModel/Actions/SupplementalPack/InstallSupplementalPackAction.cs

```diff
@@ -73,15 +78,20 @@ private void InstallSuppPack(Host host, VDI vdi)
             }
             catch (Failure failure)
             {
-                log.WarnFormat("Plugin call install-supp-pack.install({0}) on {1} failed with {2}", vdi.uuid, host.Name,
-                               failure.Message);
-                throw;
+                log.WarnFormat("Plugin call install-supp-pack.install({0}) on {1} failed with {2}", vdi.uuid, host.Name, failure.Message);
```

Comment:
I think this and the following log entries made more sense as Errors, not just warnings.